site-service/src/main/java/uk/ac/ox/ndph/mts/site_service/repository/SiteStore.java

```diff
@@ -11,13 +11,14 @@
 import uk.ac.ox.ndph.mts.site_service.model.Site;
 
 import java.util.List;
+import java.util.Optional;
 import java.util.stream.Collectors;
 
 /**
  * Implement an EntityStore for Site.
  */
 @Component
-public class SiteStore implements EntityStore<Site> {
+public class SiteStore implements EntityStore<String, Site> {
```

Comment:
Not a priority, but feel like the Entity=Site should be the first param.

Reply:
Was working by analogy with Map<Key,Value>. Agree that the entity type is more "important"

Reply:
Changed order.